src/elements/ngm-nadir-view.ts

```diff
@@ -64,20 +69,28 @@ export class NgmNadirView extends LitElementI18n {
       });
       this.heading = undefined;
       this.pitch = undefined;
+      this.position = undefined;
 
       if (this.height !== undefined) {
-        setCameraHeight(camera, this.height);
+        NavToolsStore.setCameraHeight(this.height);
       }
       this.height = undefined;
     } else {
       if (this.viewer.scene.cameraUnderground) {
         this.height = camera.positionCartographic.height;
-        setCameraHeight(camera, 10000);
+        NavToolsStore.setCameraHeight(10000);
       }
 
       this.pitch = camera.pitch;
       this.heading = camera.heading;
+      const targetPoint = getTargetParam();
+      if (targetPoint) {
+        this.position = Cartesian3.clone(camera.positionWC, new Cartesian3());
+        updateHeightForCartesianPositions([targetPoint], camera.positionCartographic.height, undefined, true);
+      }
       this.viewer.camera.setView({
+        destination: targetPoint,
+        convert: true,
```

Comment:
Documentation says we don't need this attribute:  optional Whether to convert the destination from world coordinates to scene coordinates (only relevant when not using 3D). Defaults to true.

Reply:
same here, thanks